server/service/devices.go

```diff
@@ -529,34 +530,37 @@ func (svc *Service) GetDeviceMDMAppleEnrollmentProfile(ctx context.Context) ([]b
 		return nil, ctxerr.Wrap(ctx, fleet.NewPermissionError("forbidden: only device-authenticated hosts can access this endpoint"))
 	}
 
-	appConfig, err := svc.ds.AppConfig(ctx)
+	cfg, err := svc.ds.AppConfig(ctx)
```

Comment:
Nit: I don't know if we have any general pattern, but I typically name this "ac" or "appConfig" to distinguish from the server config.

Reply:
good point, this is straight copy pasta from another method. I'll see if I can find all of them in a separate PR